juju/home.go

```diff
@@ -23,5 +26,20 @@ func InitJujuXDGDataHome() error {
 	if err := ssh.LoadClientKeys(osenv.JujuXDGDataHomePath("ssh")); err != nil {
 		return errors.Annotate(err, "cannot load ssh client keys")
 	}
+
+	base := osenv.JujuXDGDataHomePath("x509")
```

Comment:
are these bits needed any more? certs are loaded in the add-machine command (which is preferable, since it doesn't affect every command) -- does it need to be here too?